client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -39,23 +38,26 @@ const GoalsStep: Step = ( { navigation } ) => {
 	const translate = useTranslate();
 	const welcomeText = translate( 'Welcome!' );
 	const whatAreYourGoalsText = translate( 'What are your goals?' );
-	const headerText = (
-		<>
-			{ welcomeText } <br /> { whatAreYourGoalsText }
-		</>
-	);
 	const subHeaderText = translate( 'Tell us what would you like to accomplish with your website.' );
+	const displayAllGoals = isEnabled( 'signup/goals-step-2' );
```

Comment:
This definition can go outside of the component

Reply:
Fixed!